starter-repos/anthos-config-management/namespaces/app-environment/rolebinding.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: operator
+  namespace: app-environment
+subjects:
+  - kind: User
+    name: alice-the-app-operator@foo-corp.com
```

Comment:
This needs to be a real user name. What's the flow for updating this in the Root Repo? Does this rolebinding with the fake account get synced to the cluster immediately?

Reply:
Yes it is synced to the cluster immediately. We need to add a manual step for users between the CI/CD setup and `appctl init`. The manual step is to update the name here in the rolebinding. If we don't include this rolebinding file, users need to manually add one.

Reply:
SG this seems fine for now.